.changeset/five-chairs-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clack/prompts": minor
+---
+
+Add support for customizable spinner cancel and error messages. Users can now customize these messages either per spinner instance or globally via the new `updatePromptsSettings` function to support multilingual CLIs.
```

Comment:
Need to update this changeset with the proper example!

Reply:
will do it!